finalPrep/modZeroHero.js

```diff
@@ -74,11 +118,44 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age){
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName(){
+    console.log(this.name);
+  }
+  maximizeEnergy(){
+    this.energyLevel = 1000;
+  }
+  gainPower(number){
+    this.powerLevel + number;
```

Comment:
If you want to also assign the new value to `this.powerLevel`, you would need to write something like:

```
this.powerLevel = this.powerLevel + number;
```

or 

```
this.powerLevel += number;
```